gateway/api_loader.go

```diff
@@ -104,22 +105,33 @@ func fixFuncPath(pathPrefix string, funcs []apidef.MiddlewareDefinition) {
 	}
 }
 
-func (gw *Gateway) generateSubRoutes(spec *APISpec, subRouter *mux.Router, logger *logrus.Entry) {
+func (gw *Gateway) generateSubRoutes(spec *APISpec, router *mux.Router, logger *logrus.Entry) {
```

Comment:
The function name is `generatSubRoutes`, it takes `subRouter` as a param so it can create. The param name `subRouter` has a purpose there. I think we should not change that.

Reply:
Invalid, fixes [stutter](https://sben.dev/stutter/), takes single router so there's no ambiguity.